### ISSUE TITLE
Can I use it without GPU?

### ISSUE THREAD

**Issue by itsdapi** [jaywalnut310/vits#84]
My server haven't install a GPU yet. And all I got is just a GT710. I don't know how bad it will do.

**Comment by itsdapi** [comment]
Oh I have a pre-train model ready to use

**Comment by nikich340** [comment]
Yes, you can use it on CPU (RAM), just add .cpu() command to some lines in script (which will give your error).

**Comment by itsdapi** [comment]
> Yes, you can use it on CPU (RAM), just add .cpu() command to some lines in script (which will give your error).

Yes and it is pretty damn fast xD
